server/public/index.html

```diff
@@ -3,7 +3,11 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Final Project</title>
+  <meta property="og:image" content="https://chalk-up-app.herokuapp.com/server/public/images/mega-image.jpg">
```

Comment:
You should remove `server/public` from here. `server/public` is the root of your server (using static middleware) so all files and directories in there are accessible from `https://chalk-up-app.herokuapp.com/`

Your og image url should be

```
https://chalk-up-app.herokuapp.com/images/mega-image.jpg
```